### ISSUE TITLE
Military Power

### ISSUE THREAD

**Issue by UniversalRed** [VictoriaUltimate/Victoria-Ultimate#2]
In the current state of things, military power is quite a bothersome issue. The AI USA alone can have a military score of 50,000 by the end of 2030 if you were to start in the 2000 bookmark. In addition, because of the amount of armies being moved simultaneously, it causes the game to run slow even if you have optimized settings. There should be a solution like increasing the price of raising a brigade and having consumption up even higher in order to balance the game off a little.

**Comment by Palisight** [comment]
In your opinion, do you think this is partially a population growth problem (i.e. is the situation fine in 2000, but deteriorates over time due to population growth), or do you think its just a problem that units are too cheap to raise/maintain and countries can build too many?

**Comment by UniversalRed** [comment]
It's actually a mix of both. The issue seems to become problematic over a course of 10 - 20 years after starting a game in the 2000/2023 bookmark. Also, countries like America can easily create large quantities of armies because of their economy tying into their military and the relatively cheap military goods. 

   So, I believe that there should be global events relating to low population growth for countries that are fully developed and also trying to raise the factory cost of industries for countries that do extremely well at tying their economy with their military. In other words, we'll probably need to implement a system for countries so that they can be categorized as first world, second world or third world countries and then tying events into these categorized countries. Just a thought but it might balance the game play for the game in the long run

**Comment by Palisight** [comment]
I think the US population at the start of the 2000 scenario is fine. I'm guessing if the population gets out of control in a few decades, its probably due to the known issue of too much immigration to the US.

As for 1st/2nd/3rd world country balancing, I guess the issue is how is it to be done. 1st world Laissez Faire countries like Hong Kong or Singapore for example should not be given economic penalties.

If we want to represent how the British Empire became over-extended, and stopped being the dominant economic power in the world, we probably could do that with increased military costs

In the future, we will say the same thing about the US, but it hasn't happened yet because no suitable "replacement world power" has been found (China the assumed replacement is not ready yet for example)

**Comment by UniversalRed** [comment]
The US population is fine, but we'll probably have to do something about the immigration problem. Maybe creating an event that limits their immigration by 25% would work but that will have to wait.

For the 1st/2nd/3rd world balancing, it should be relevant to the Cold War, as they were categorized by the West and East blocs. also we could try to make country specific events for countries like the U.K. through these different events. 

Or, what we could do is create developed, developing and undeveloped countries, though this would only be the case for countries in the 2000, 2023 bookmark. The only benefit to this is that we can then categorize developed countries (excluding Hong Kong/Singapore) so that they have less population growth because of the advanced education and health care, as well as maybe a higher factory cost to limit their economic superiority.

**Comment by Palisight** [comment]
The issue with specific country events is it ignores how the game is going. If for example France wins the Napoleonic wars, France in that game will take the place the British Empire did in the Victorian era. 

If the Polish-Lithuanian Commonwealth or the Ming Empire survives to the 1900s, the game should punish those countries if they over-extend to the same extent the British Empire did. 

If France wins the Seven Years War, France should have to deal with the possibility of North American colony revolts.

Basically my point is we should be focusing on why 1st world countries develop, and how to model it in the game. 

For example you suggested a correlation between education (or in game terms literacy) and population growth. I'm not sure why health care would reduce population growth (I would think it would increase it)

As for your increased factory cost idea, maybe some social reforms should increase the cost of factories for that nation.

**Comment by UniversalRed** [comment]
This is a larger idea than what I had originally thought. If we are to do this idea then we might have to make guidelines for countries to be classified as a empire, instead of just focusing on historical countries that created an empire. I'll jot down notes to figure out a guideline for how this system will work, and then I'll show it to you once it's figured out.

**Comment by UniversalRed** [comment]
https://gyazo.com/c50e9eedad559411474c546efbc995c0

This is a rough guideline for what the different country statuses might look like unless we change things (which we probably will). If you want the actual file to make changes I can send it to you if you want.

**Comment by Palisight** [comment]
1.) To the best of my knowledge, there is no event condition related to industry score, thus that can't be a criterion.

2.) Scripting based on 10 social reforms would be a nightmare. It also would be weird to say the US for example is a developing country at the start of the 2000 scenario for example because the only publically available healthcare would be medicaid and medicare. Making it based on 7/10 for example would require so many AND and OR conditions scripting that it would be a nightmare.

3.) Why should developed countries not be susceptible to religious conflicts or corruption? (I'm also at a loss to determine criterions for whether or not a country is corrupt or not). V2 already handles bad effects from poverty very well, and modelling decentralization pretty much has to be coded in a case by case basis in the form of substates and vassals.

4.) Consequences from horrible labour laws can be handled based on social reforms instead of a developing country modifier. Maybe pollution can be handled from the safety regulation reform in a similar manner.

5.) Investing in developing countries or providing humanitarian aid would have to be coded on a case by case basis. Basically I'm not seeing a need for the developed country modifier either

6.) The game already handles events promoting social reforms. We could make those events more or less likely to fire based on whether or not the country has researched specific technologies. 

7.) Pensions and unemployment subsidies do benefit the working class in V2. Unless you are implying it should be different for non modernized economies, which would require .exe modding, which is illegal

8.) Raising troop/naval costs can be done through the tech tree. Higher tech units can have higher supply consumption. Units that are unlocked at higher techs can be made more expensive.

Basically I'm not seeing the point of these country statuses. You have good ideas here about things that need to be added, but I'm thinking they should be triggered based off of reform level or tech level instead country statuses

9.) Countries that aren't at least secondary powers don't typically have overseas provinces

**Comment by UniversalRed** [comment]
I see your points. I guess we should come back to this at another time then. But then isn't there a way to limit the AI's production of armies through some sort of custom decision?

**Comment by Palisight** [comment]
Minimum and maximum military spending can be regulated by political parties and/or country modifiers

Since military spending affects the promotion and demotion of soldier pops, the % of soldier pops can be regulated this way. Sadly as PI did not include sailor pops, this won't regulate the amount of ships

That being said, I'm thinking the immigration issue should be fixed first before trying to regulate the % of soldier pops

**Comment by UniversalRed** [comment]
Let's see what we can do then before coming back to this.
